examples/python/Quantinuum_conditional_gates.py

```diff
@@ -0,0 +1,65 @@
+# <div style="text-align: center;">
+# <img src="https://assets-global.website-files.com/62b9d45fb3f64842a96c9686/62d84db4aeb2f6552f3a2f78_Quantinuum%20Logo__horizontal%20blue.svg" width="200" height="200" /></div>
+
+# # Conditional Gates
+
+# Quantinuum H-Series hardware and pytket support conditional gating. This may be for implementing error correction or reducing noise. This capability is well-supported by Quantinuum hardware, which supports mid-circuit measurement and reset and qubit reuse. See [Conditional Gates](https://cqcl.github.io/pytket/manual/manual_circuit.html#classical-and-conditional-operations) for more information on pytket's implementation. The following example demonstrates a quantum teleportation protocol.
+
+from pytket.circuit import Circuit, if_bit
+from pytket.circuit.display import render_circuit_jupyter
+
+# create a circuit and add quantum and classical registers
+circ = Circuit(name="Conditional Gates Example")
+qreg = circ.add_q_register("q", 3)
+creg = circ.add_c_register("b", 2)
+
+# prepare q[0] to be in the state |->, which we wish to teleport to q[2]
+circ.X(qreg[0]).H(qreg[0])
+
+# prepare a Bell state on qubits q[1] and q[2]
+circ.H(qreg[1])
+circ.CX(qreg[1], qreg[2])
+
+# construct the teleportation protocol
+circ.CX(qreg[0], qreg[1])
+circ.H(qreg[0])
+circ.Measure(qreg[0], creg[0])
+circ.Measure(qreg[1], creg[1])
+
+# if (creg[1] == 1)
+circ.X(qreg[2], condition=if_bit(creg[1]))
+
+# if (creg[0] == 1)
+circ.Z(qreg[2], condition=if_bit(creg[0]))
+
+render_circuit_jupyter(circ)
+
+# We can utilise pytket's [Assertion](https://cqcl.github.io/pytket/manual/manual_assertion.html#assertion) feature to verify the successful teleportation of the state $| - \rangle$.
```

Comment:
The link to the assertion section of the manual is now

https://tket.quantinuum.com/user-manual/manual_assertion.html

Reply:
Updated